README.md

```diff
@@ -149,70 +127,21 @@ Some examples:
 
 # capture 4mb of video from camera
 ./ankerctl.py pppp capture-video -m 4mb output.h264
-```
-
-## Webserver
 
-ankerctl can also be used as a webserver to allow slicers like prusaslicer to print directly to the printer. 
-
-![Screenshot of prusa slicer](/static/img/setup/prusaslicer-2.png?raw=true "Screenshot of prusa slicer")
-
-To start the webserver run the following command, then navigate to [http://localhost:4470](http://localhost:4470)
-
-```sh
-./ankerctl.py webserver run
+# select printer to use when you have multiple
+./ankerctl.py -p <index> #index starts at 0 and goes up to the number of printers you have
 ```
 
-You can alternativly use docker compose to start the webserver running behind nginx
-
-```sh
-docker compose up
-```
-
-
+### Printing Directly from PrusaSlicer
 
-## Docker
-
-While running the python script is generally prefered, there may be situations where you want a more portable solution. For this, a docker image is provided.
-
-```sh
-docker build -t ankerctl .
-```
-
-Example usage (no peristent storage)
-```bash
-docker run \
-  -v "$HOME/Library/Application Support/AnkerMake/AnkerMake_64bit_fp/login.json:/tmp/login.json" \
-  ankerctl config decode /tmp/login.json
-```
-
-Example usage (with peristent storage)
-```bash
-# create volume where we can store configs
-docker volume create ankerctl_vol
-
-# generate /root/.config/ankerctl/default.json which is mounted to the docker volume
-docker run \
-  -v ankerctl_vol:/root/.config/ankerctl \
-  -v "$HOME/Library/Application Support/AnkerMake/AnkerMake_64bit_fp/login.json:/tmp/login.json" \
-  ankerctl config import /tmp/login.json
-
-# Now that there is a /root/.config/ankerctl/default.json file that persists in the docker volume
-# we can run ankerctl without having to specify the login.json file
-docker run \
-  -v ankerctl_vol:/root/.config/ankerctl \
-  ankerctl config show
-```
+ankerctl can allow slicers like PrusaSlicer (and its derivatives) to send print jobs directly to the printer using the slicer’s built in communications tools. The web server must be running in order to send jobs to the printer. Currently there’s no way to store the jobs for later printing on the printer, so you’re limited to using the “Send and Print” option only to immediately start the print once it’s been transmitted. Additional instructions can be found in the web interface.
```

Comment:
As a general note, all lines were un-wrapped from the previous version.

I'm not religious about it, and it's consistent, so that's fine. But this single line is hundreds of columns long, so it's leading to some hefty diffs, and harder reviews.

Can we either word-wrap at some consistent column (like before), or just split up the longest sentences into a line per paragraph?

Either way is fine

Reply:
Cleaned up by splitting longest sentences into a line per paragraph.